ui/rollup.config.js

```diff
@@ -0,0 +1,38 @@
+import merge from 'deepmerge';
+import resolve from '@rollup/plugin-node-resolve';
+import copy from 'rollup-plugin-copy';
+
+// use createSpaConfig for bundling a Single Page App
+import {createSpaConfig} from '@open-wc/building-rollup';
+// use createBasicConfig to do regular JS to JS bundling
+// import { createBasicConfig } from '@open-wc/building-rollup';
+
+// TODO: make SPA decision and cleanup this config further
```

Comment:
Assign an owner for TODO

Reply:
Would it be better to assign an issue number to the TODO? And have a list of open issues as trackers?

Reply:
assigned to myself for now, not sure if github does automated issue creation/tracking but I know a todo bot exists